### ISSUE TITLE
Query: Negated Fields causing an error in Node/C++

### ISSUE THREAD

**Issue by caioliberali** [tree-sitter/node-tree-sitter#84]
When using the Playground the query works normally but when using Node with C++ parser the same query causes an error ( if I remove the Negated Field it works ).

**Example:**
[https://tree-sitter.github.io/tree-sitter/playground](https://tree-sitter.github.io/tree-sitter/playground)

**Code**
```cpp
class MyClass
{
public:
	MyClass();
	MyClass(int b){}

	void Function();
	void FunctionB(){}
}
```

**Query**
`[
	(declaration declarator: (function_declarator declarator: (identifier) @N)) @C
        (function_definition !type declarator: (function_declarator declarator: [ (identifier) @N (field_identifier) @N ] )) @C
]
`

**!type** is the keyword that is causing an error.

Can anybody help me?
Thank you!

**Comment by ahlinc** [comment]
@caioliberali it's because negated patterns were added in the core tree-sitter library [after v0.19.3](https://github.com/tree-sitter/tree-sitter/commit/bc0ef5f3736453d615e7ee1ed9e394d0631e8b4f) version and doesn't released yet even in the core library. Parsers in the playground were updated [5 days ago.](https://github.com/tree-sitter/tree-sitter.github.io/commit/b0b6969ce8e9d7df2fb0524176782d4ae680e857) but this nodejs binding was released only for v0.19.0 [9 days ago](https://github.com/tree-sitter/node-tree-sitter/releases/tag/v0.19.0).

In this situation you have one option to:
1. Clone this repo.
2. Update the core tree-sitter library in the [vendor/tree-sitter](https://github.com/tree-sitter/node-tree-sitter/tree/master/vendor) to the master.
3. Compile this binding.
4. Install this compiled version to your node_modules.

**Comment by caioliberali** [comment]
@ahlinc Thank you very much again!!
